cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -614,6 +614,42 @@ def _validate(self, name: str):
             self._add_failure(f"It is forbidden to use '{name}' as a name.", FailureLevel.ERROR)
 
 
+class SharedStorageMountDirValidator(Validator):
+    """
+    Shared storage mount directory validator.
+
+    Make sure the mount directory is not the same as any reserved directory.
+    """
+
+    def _validate(self, mount_dir: str):
+        reserved_directories = [
+            "/bin",
+            "/dev",
+            "/home",
+            "/lib64",
+            "/opt",
+            "/root",
+            "/sbin",
+            "/sys",
+            "/usr",
+            "/boot",
+            "/etc",
+            "/lib",
+            "/proc",
+            "/run",
+            "/srv",
+            "/var",
+        ]
+        default_directories = ["/scratch"]
+        if not mount_dir.startswith("/"):
+            mount_dir = "/" + mount_dir
+        if mount_dir in reserved_directories + default_directories:
+            self._add_failure(
+                f"Error: The shared storage mount directory {mount_dir} is reserved. " "Please use another directory",
```

Comment:
minor: autoformat created an extra space in the middle.

Reply:
Done